kolibri/plugins/learn/assets/src/vue/learn-content-page.vue

```diff
@@ -33,6 +33,7 @@
 
 <style lang="stylus" scoped>
 
+  @import 'jeet'
   @require '~core-theme.styl'
```

Comment:
Any reason to choose `import` instead of `require`?

I'm not very familiar with them, but [the docs say](http://stylus-lang.com/docs/import.html#require) that `require` is like `import` but only imports the file once.